pandas/io/clipboard.py

```diff
@@ -75,21 +74,22 @@ def to_clipboard(obj, excel=None, sep=None, **kwargs):  # pragma: no cover
       - OS X:
     """
     from pandas.util.clipboard import clipboard_set
-    if excel is None:
-        excel = True
 
-    if excel:
-        try:
-            if sep is None:
-                sep = '\t'
-            buf = StringIO()
-            obj.to_csv(buf, sep=sep, **kwargs)
-            clipboard_set(buf.getvalue())
-            return
-        except:
-            pass
-
-    if isinstance(obj, DataFrame):
+    # COMPAT: allow None for `excel` and `sep`
+    if excel in (True, None) and isinstance(obj, (DataFrame, Series)):
+        if 'sep' not in kwargs or not kwargs['sep']:
```

Comment:
I tried to make this compatible with current `to_clipboard` behaviour and its tests. 

1. `excel=None` it's treated as `True`
2. `sep=None` is ignored

'Cause i thought it's better to pass `sep` as a `kwargs` item i have to check for its existense&value. Is it better to convert it back to explicit function argument?